packages/server/src/user/user.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Mutation, Args, Resolver, Query } from '@nestjs/graphql';
+import { UserService } from './user.service';
+import { UserModel } from './model/user.model';
+import { User } from '@prisma/client';
+
+@Resolver()
+export class UserResolver {
+  constructor(private readonly userService: UserService) {}
+
+  @Query(() => UserModel)
+  async listUsers(): Promise<UserModel[]> {
```

Comment:
We probably want to add in an @Project Decorator here to filter by project id, We probably only want to display users for a given project and not all users. 

We probably need to create that decorator, for now add a param for project id to be added